### ISSUE TITLE
Build ops for optimizing the dynamics model parameters

### ISSUE THREAD

**Issue by thiagopbueno** [thiagopbueno/model-aware-policy-optimization#11]
Build ops for implementing a step of gradient descent over the dynamics model parameters given a TensorFlow optimizer and a batch of experiences.

**Comment by 0xangelo** [comment]
Closed by #48 and #60
